README.md

```diff
@@ -100,6 +101,26 @@ General security best practices apply:
   * Ultimately, the security of a system depends on the capabilities of its administrator.
   * Care should be taken when installing new software; only install from official sources that the developers indicate on their official website/github/etc.
 
+# Threat Modeling
+
+The first and most important step for security and privacy is to create a threat model. After all, how can we have security when we don't even know what we're securing against? Each person will have their own needs so everyone's threat model will be different. Threat models tend to evolve over time as our situation changes, so be sure to periodically reassess your threat model.
+
+Step 1: Identify what you're protecting
+
+This is probably a lot of things: your phone, your laptop, passwords stored on your devices, internet browsing history, etc. Make a list of all the things you can think of that you want to protect and store it somewhere.
```

Comment:
Let's be more specific here. Perhaps something like, identify what data is *valuable*, maybe with three or so tiers of data sensitivity classification (public, sensitive, secret), and draw rules around that. What are the so-called "crown jewels" on the system which are most important to defend?

Reply:
Can you give an example? Not really sure what you mean with the different sensitivity levels.

Reply:
Ok tried to incorporate that